packages/next/client/page-loader.js

```diff
@@ -5,6 +5,19 @@ import { getRouteMatcher } from './../next-server/lib/router/utils/route-matcher
 import { getRouteRegex } from './../next-server/lib/router/utils/route-regex'
 import { delBasePath } from './../next-server/lib/router/router'
 
+let trustedTypesPolicy = undefined
+if (window?.trustedTypes?.createPolicy) {
+  trustedTypesPolicy = window.trustedTypes.createPolicy('next-trusted-types', {
+    // Needs security review regarding DOM XSS
+    createHTML(dirty) {
```

Comment:
is this even needed for this particular case

Reply:
Looks like it isn't used, so the createHTML method here can be dropped.